pkg/gateway/kubernetes.go

```diff
@@ -11,21 +11,23 @@ import (
 	"strings"
 )
 
-type ServiceMapperConfig struct {
+type kubernetesReverseProxyConfigGetterConfig struct {
```

Comment:
I feel like this annotation prefix should likely be the same regardless of where it's coming from in Kube. Could it just be part of a KubernetesConfig?